Formula/snapcraft.rb

```diff
@@ -218,8 +218,8 @@ class Snapcraft < Formula
   end
 
   resource "simplejson" do
-    url "https://files.pythonhosted.org/packages/98/87/a7b98aa9256c8843f92878966dc3d8d914c14aad97e2c5ce4798d5743e07/simplejson-3.17.0.tar.gz"
-    sha256 "2b4b2b738b3b99819a17feaf118265d0753d5536049ea570b3c43b51c4701e81"
+    url "https://files.pythonhosted.org/packages/49/45/a16db4f0fa383aaf0676fb7e3c660304fe390415c243f41a77c7f917d59b/simplejson-3.17.2.tar.gz"
+    sha256 "75ecc79f26d99222a084fbdd1ce5aad3ac3a8bd535cd9059528452da38b68841"
   end
```

Comment:
According to the [`requirements.txt`](https://github.com/snapcore/snapcraft/blob/e4e08797fb4e2d5110504d46bed39ee00382a653/requirements.txt#L20) `simplejson==3.8.2` is required.

Clearly, this had been updated from that point before. My guess is that it will continue to work fine. It seems like the `requirements.txt` uses `==` everywhere even though most could probably be `>=`

Reply:
very good call out.